addons/website/static/src/snippets/s_dynamic_snippet/options.js

```diff
@@ -225,6 +232,13 @@ const dynamicSnippetOptions = options.Class.extend({
             }
             selectUserValueWidgetElement.appendChild(button);
         }
+        if (!Object.keys(data).length) {
+            const button = document.createElement('we-button');
+            button.dataset.selectDataAttribute = 'empty';
+            button.dataset.emptyListPlaceholder = true;
+            button.innerText = '/';
+            selectUserValueWidgetElement.appendChild(button);
```

Comment:
The problem is the same in 14.0/15.0, no? So yes, I would make another PR for this commit, targeting 14.0 :+1: 

I would fix this differently:
- Not link it to the option/selectDataAttribute, this should be a feature at the UserValueWidget level
- Can't we have a pure CSS solution? (something like we-select:empty::before { content: "/"; .... };